<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PtzCtrlWnd</class>
 <widget class="QWidget" name="PtzCtrlWnd">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>547</width>
    <height>354</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PtzCtrlWnd</string>
  </property>
  <layout class="QVBoxLayout" name="mainLayout" stretch="10,1">
   <property name="bottomMargin">
    <number>30</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="ptzLayout" stretch="5,1">
     <property name="leftMargin">
      <number>20</number>
     </property>
     <property name="topMargin">
      <number>10</number>
     </property>
     <property name="rightMargin">
      <number>20</number>
     </property>
     <item>
      <layout class="QVBoxLayout" name="ptzctrlLayout"/>
     </item>
     <item>
      <layout class="QVBoxLayout" name="zoomLayout">
       <item>
        <widget class="QPushButton" name="btnZoomIn">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>40</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>40</width>
           <height>40</height>
          </size>
         </property>
         <property name="baseSize">
          <size>
           <width>40</width>
           <height>40</height>
          </size>
         </property>
         <property name="text">
          <string>In</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnZoomout">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>40</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>40</width>
           <height>40</height>
          </size>
         </property>
         <property name="text">
          <string>Out</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="speedLayout" stretch="0,0,0">
     <property name="leftMargin">
      <number>10</number>
     </property>
     <property name="rightMargin">
      <number>10</number>
     </property>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>云台速度</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="speedSlider">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lblSliderValue">
       <property name="text">
        <string>1</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
